src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -48,33 +47,34 @@
 #?(:cljs
 (defmethod ct/report [::chuck-capture :error]
   [m]
-  (swap! *chuck-captured-reports* conj m)))
+  (swap! *chuck-captured-reports* conj (assoc m ::testing-contexts (:testing-contexts (ct/get-current-env))))))
 
 #?(:cljs
 (defmethod ct/report [::chuck-capture :fail]
   [m]
-  (swap! *chuck-captured-reports* conj m)))
+  (swap! *chuck-captured-reports* conj (assoc m ::testing-contexts (:testing-contexts (ct/get-current-env))))))
 
 #?(:cljs
 (defmethod ct/report [::chuck-capture :pass]
   [m]
-  (swap! *chuck-captured-reports* conj m)))
+  (swap! *chuck-captured-reports* conj (assoc m ::testing-contexts (:testing-contexts (ct/get-current-env))))))
 
 (defn capture-reports*
   [reports-atom f]
   #?(:clj
-     (binding [ct/report #(swap! reports-atom conj %)]
+     (binding [ct/report #(swap! reports-atom conj (assoc % ::testing-contexts ct/*testing-contexts*))]
        (f))
 
      :cljs
      (binding [*chuck-captured-reports* reports-atom
-               cljs.test/*current-env* (cljs.test/empty-env ::chuck-capture)]
+               ct/*current-env* (assoc (ct/empty-env ::chuck-capture)
+                                       :testing-contexts (:testing-contexts (ct/get-current-env)))]
```

Comment:
is one of these keywords supposed to be namespaced? I'm comparing to the (new) line 112 in particular

Reply:
Good question, I don't think so.

This forwards the current `:testing-contexts` to (new) lines 50, 55, 60, 65 so then we can attach them to the report via `::testing-contexts`. Then, `::testing-contexts` is retrieved in `-report`.

The same idea is done for `:clj` on line `65`.

Reply:
To be clearer, these cannot be namespaced because it's working with envs from `cljs.test`, which uses `:testing-contexts` as a field.